### PR TITLE
asuka: init at 0.8.0

### LINKED ISSUE #87783: Package request: asuka
**Project description**
A Gemini client written in Rust with NCurses.

**Metadata**

* source URL: https://git.sr.ht/~julienxx/asuka
* license: mit
* platforms: unix, linux, darwin

### PR DESCRIPTION
**Pull request by sikmir**
###### Motivation for this change
[**asuka**](https://git.sr.ht/~julienxx/asuka) - Gemini Project client written in Rust with NCurses.
Resolves https://github.com/NixOS/nixpkgs/issues/87783.

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

asuka

### CODE REVIEW

File: pkgs/applications/networking/browsers/asuka/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  src = fetchurl {
+    url = "https://git.sr.ht/~julienxx/${pname}/archive/${version}.tar.gz";
+    sha256 = "10hmsdwf2nrsmpycqa08vd31c6vhx7w5fhvv5a9f92sqp0lcavf0";
+  };
+
+  cargoPatches = [ ./cargo-lock.patch ];
+
+  cargoSha256 = "0csj63x77nkdh543pzl9cbaip6xp8anw0942hc6j19y7yicd29ns";
+  verifyCargoDeps = true;
```
Comment (danieldk) [inline pkgs/applications/networking/browsers/asuka/default.nix:15]:
```suggestion
```

`verifyCargoDeps` is no longer used by `buildRustPackage`.

File: pkgs/applications/networking/browsers/asuka/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  nativeBuildInputs = [ pkgconfig ];
```
Comment (danieldk) [inline pkgs/applications/networking/browsers/asuka/default.nix:17]: Small nitpick: add new line between `nativeBuildInputs` and `buildInputs`.

**Review by danieldk — CHANGES_REQUESTED** [review]
Looks good, just two small nitpicks.

File: pkgs/applications/networking/browsers/asuka/default.nix (hunk 1 above)
Comment (sikmir) [reply pkgs/applications/networking/browsers/asuka/default.nix]: Removed.

File: pkgs/applications/networking/browsers/asuka/default.nix (hunk 2 above)
Comment (sikmir) [reply pkgs/applications/networking/browsers/asuka/default.nix:16]: Fixed.

**Review by danieldk — APPROVED** [review]
LGTM. Thanks!

Result of `nixpkgs-review pr 91453` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- asuka
</details>

Binary works as expected.

**Pull request merged**